### ISSUE TITLE
SMB 1.0/CIFS File Sharing Support installed by default

### ISSUE THREAD

**Issue by aakrol** [MicrosoftDocs/windowsserverdocs#694]
On a VM deployed on Azure on 25/04/2018, the SMB 1.0/CIFS File Sharing Support role is no longer installed by default.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e5acee3a-b458-172f-127d-07e0406bd46f
* Version Independent ID: a11c4876-e20a-4307-b1b7-38f414a4532c
* Content: [Windows Server 2016 Products and Editions](https://docs.microsoft.com/en-us/windows-server/get-started/2016-edition-comparison)
* Content Source: [WindowsServerDocs/get-started/2016-Edition-Comparison.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/get-started/2016-Edition-Comparison.md)
* Service: **unspecified**
* GitHub Login: @jaimeo
* Microsoft Alias: **jaimeo**

**Comment by lizap** [comment]
@jaimeo - can you take a look at this one? Thanks!

**Comment by pedervm** [comment]
SMB 1 is not installed as default, since build 1709, so it shouldn't be installed on Azure as default:

https://support.microsoft.com/en-us/help/4034314/smbv1-is-not-installed-by-default-in-windows

**Comment by jaimeo** [comment]
Confirming with SME.

**Comment by jaimeo** [comment]
Sorry for the delay--I was researching this thoroughly. The Windows Server 2016 edition comparison topic is correct: SMB1 *IS* installed by default in Windows Server 2016, which is the release that the topic applies to. (If it also applied to Windows Server, version 1709, that would be listed in the "applies to" section near the top.) 

However, we did miss that it had been removed as pre-installed feature in Windows Server, version 1709. I have updated https://docs.microsoft.com/en-us/windows-server/get-started/removed-features-1709 to reflect this, including a link to the KB article that goes into detail about it. Thanks for helping us with this!
